editor/js/ui/library.js

```diff
@@ -447,7 +447,9 @@ RED.library = (function() {
                             click: function() {
                                 //TODO: move this to RED.library
                                 var flowName = $("#node-input-library-filename").val();
-                                if (!/^\s*$/.test(flowName)) {
+                                if(flowName.split(/[\\/]/).pop() === "" || /^\s*$/.test(flowName)) {
```

Comment:
Can I suggest the following test instead? Doesn't require mental parsing of regex to understand the intent of the test.

```suggestion
                                flowName = flowName.trim();
                                if(flowName === "" || flowName.endsWith("/")) {
```